app/_kong_plugins/datakit/index.md

```diff
@@ -0,0 +1,1576 @@
+`headers` input).
+
+The `request.body` and `service_response.body` outputs have a similar behavior.
+If the corresponding `Content-Type` header matches the JSON mime-type, the
+`body` output is automatically JSON-decoded.
+
+### `call` node
+
+Send an HTTP request and retrieve the response.
+
+#### Inputs
+
+* `body`: Request body
+* `headers`: Request headers
+* `query`: Key-value pairs to encode as the request query string
+
+#### Outputs
+
+* `body`: The response body
+* `headers`: The response headers
+* `status`: The HTTP status code of the response
+
+#### Configuration attributes
+
+* `url` (**required**): The URL
+* `method`: The HTTP method (default is `GET`)
+* `timeout`: The dispatch timeout, in milliseconds
+
+#### Examples
+
+Make an external API call:
+
+```yaml
+- name: CALL
+  type: call
+  url: https://example.com/foo
+```
+
+Send a POST request with a JSON body:
+
+```yaml
+- name: POST
+  type: call
+  url: https://example.com/create-entity
+  method: POST
+  inputs:
+    body: ENTITY
+
+- name: ENTITY
+  type: static
+  values:
+    id: 123
+    name: Datakit
+```
+
+#### Automatic JSON body handling
+
+If the data connected to the `body` input is an object, it will automatically be
+encoded as JSON, and the request `Content-Type` header will be set to
+`application/json` unless already present in the `headers` input.
+
+Similarly, if the response `Content-Type` header matches the JSON mime-type, the
+`body` output will be JSON-decoded automatically.
+
+#### Async execution
+
+This is an `async` node. This means that the request will be sent in the
+background while Datakit executes any other nodes (save for any nodes which
+depend on it). Multiple call nodes are executed concurrently when no dependency
+order enforces it.
+
+In this example, both `CALL_FOO` and `CALL_BAR` will be started as soon as
+possible, and then Datakit will block until both have finished to run
+`JOIN`:
+
+```yaml
+- name: CALL_FOO
+  type: call
+  url: https://example.com/foo
+
+- name: CALL_BAR
+  type: call
+  url: https://example.com/bar
+
+- name: JOIN
+  type: jq
+  jq: "."
+  inputs:
+    foo: CALL_FOO.body
+    bar: CALL_BAR.body
+```
+
+#### Failure conditions
+
+The `call` node fails execution if a network-level error is encountered or if
+the endpoint returns a non-2xx status code. It will also fail if the endpoint
+returns a JSON mime-type in the `Content-Type` header if the response body is
+not valid JSON.
+
+#### Limitations
+
+Due to platform limitations, the `call` node can't be executed after proxying a
+request, so attempting to configure the node using outputs from the upstream service
+response will yield an error:
+
+```yaml
+- name: CALL
+  type: call
+  url: https://example.com/
+  method: POST
+  inputs:
+    # dependency error!
+    body: service_response.body
+```
+
+### `jq` node type
+
+Execution of a jq script for processing JSON. See the official
```

Comment:
```suggestion
The `jq` node executes a jq script for processing JSON. See the official
```
Not sure if this is accurate, but the original sentence is incomplete (missing subject)